### DISCUSSION TITLE
Date Range picker doesn't update the date immediately

### DISCUSSION THREAD

**Discussion by hagealex** [Laravel-Backpack/community-forum#1169] [General]
Hello everyone!

I'm working with the date range picker and have one question:

When I open the calendar the default day is today, as we see in the following screenshot:

![grafik](https://github.com/user-attachments/assets/410caeb1-f59d-4478-a53b-1a2721236714)

Then, when I click on another day, the date is not updated immediately:

![grafik](https://github.com/user-attachments/assets/d23a488b-1cf5-435e-a941-d58b5e0d69e2)

Only when I click again on the same day or I click on another date for a range:

![grafik](https://github.com/user-attachments/assets/e68232c7-16a0-4efd-afc2-dc08d9f69ef0)

I would expect the day to be updated immediately, especially when I want to set only one day for the date.

Is it possible to achieve this with some configuration? Or is this behavior intended?

Thanks in advance and kind regards :)

Just for the note:

My current Backpack version is 6.7.29.

**Comment by jcastroa87** [comment]
Hello @hagealex 

This is the default behaivor of https://www.daterangepicker.com/ even in the examples you can see is needed to click "Apply" button to get the update.

As this is a "range" date, you need to select 2 values to get the range, if you need to choose the same day, is needed to select 2 times.

If you just need to select one date, you can try to play with option "singleDatePicker" as you can read in https://www.daterangepicker.com/#options

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
